melodic/distribution.yaml

```diff
@@ -8,6 +8,22 @@ release_platforms:
   ubuntu:
   - bionic
 repositories:
+  VisualStates:
+    doc:
+      type: git
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: upstream/0.2.2
+    release:
+      packages:
+      - visualstates
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: 0.2.2-1
+    source:
+      type: git
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: upstream/0.2.2
```

Comment:
The source version must be a branch not a tag so that the buildfarm can keep up. Note that this branch and the one for the doc above should be on the upstream repository not on the release repository generated tags or branches.